libs/libjson/_internal/Source/JSONNode.h

```diff
@@ -666,9 +666,9 @@ inline void JSONNode::set_name(const json_string & newname) json_nothrow{
 	   return static_cast<json_string>(*internal);
     }
 
-    inline int JSONNode::as_int(void) const json_nothrow {
+    inline json_int_t JSONNode::as_int(void) const json_nothrow {
```

Comment:
does this json_int_t have to be compatible/the same as streamoff? How do we make sure that is the case? 
streamoff will point to the largest integral type supported by language that will depend on the architecture and the compiler standard (unsigned long int or unsigned long long int)
http://en.cppreference.com/w/cpp/language/types
json_int_t might point only to unsigned long int